addons/website_sale/static/src/js/website_sale_tracking.js

```diff
@@ -57,23 +40,36 @@ publicWidget.registry.websiteSaleTracking = publicWidget.Widget.extend({
      * @private
      */
     _trackGA: function () {
-        var websiteGA = window.ga || function () {};
+        const websiteGA = window.gtag || function () {};
         websiteGA.apply(this, arguments);
     },
     /**
      * @private
      */
     _vpv: function (page) { //virtual page view
-        this._trackGA('send', 'pageview', {
-          'page': page,
-          'title': document.title,
+        this._trackGA('event', 'page_view', {
+            'page_path': page,
         });
     },
 
     //--------------------------------------------------------------------------
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     */
+     _onViewItem(event, data) {
+        this._trackGA('event', 'view_item', data);
```

Comment:
Can't work, this was sending
```javascript
{
    "item_id": 37,
    "item_name": "[DESK0005] Customizable Desk (Custom, White)",
    "item_category": "Office Furniture",
    "currency": "USD",
    "price": 750
}
```
instead of (as written in the doc https://developers.google.com/analytics/devguides/collection/ga4/reference/events#view_item)
```javascript
{
    "currency": "USD",
    "value": 750,
    "items": [
        {
            "item_id": 37,
            "item_name": "[DESK0005] Customizable Desk (Custom, White)",
            "item_category": "Office Furniture",
            "currency": "USD",
            "price": 750
        }
    ]
}
```

Or maybe it did by luck, but it is not what google expects